### PR TITLE
[v7r0] TS: __deleteTransformationFiles also attempts to remove DataFiles

### PR DESCRIPTION
**Pull request by chaen**
to fix https://github.com/DIRACGrid/DIRAC/issues/3710 because I have enough of those 8h long cleaning queries

BEGINRELEASENOTES

*TS
CHANGE: clean the DataFiles table when cleaning a Transformation

ENDRELEASENOTES

### CODE REVIEW

**Comment by fstagni** [comment]
I'll check the failing unit tests (they have nothing to do with this PR)

**Comment by atsareg** [comment]
Looks it can be approved, please do.

**Comment by fstagni** [comment]
Pleas rebase

**Comment by chaen** [comment]
done

File: tests/Jenkins/utilities.sh [hunk 1]
```diff
@@ -345,7 +345,7 @@ function installDIRAC(){
     } >> "$CLIENTINSTALLDIR/dirac-ci-install.cfg"
   fi
 
-  if ! ./dirac-install -r $DIRAC_RELEASE -t client $INSTALLOPTIONS "$CLIENTINSTALLDIR/dirac-ci-install.cfg" $DEBUG; then
+  if ! ./dirac-install -r $DIRAC_RELEASE -t client "${INSTALLOPTIONS[@]}" "$CLIENTINSTALLDIR/dirac-ci-install.cfg"; then
```
Comment (fstagni) [inline tests/Jenkins/utilities.sh:348]: Why did you remove the `$DEBUG` flag?

Comment (chaen) [reply tests/Jenkins/utilities.sh:348]: because it is already in the `INSTALLOPTIONS` array
